### ISSUE TITLE
Subcutaneous Panniculitis-Like T-Cell Lymphoma

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7748]
**Preferred term label:**
Subcutaneous Panniculitis-Like T-Cell Lymphoma

**Synonyms**

**Definition (free text, please give PubMed ID)**
A form of T-cell non-Hodgkin lymphoma in which cytotoxic CD8+ T cells infiltrate adipose tissue forming subcutaneous nodules.

**Parent term (use hpo.jax.org/app)**
T-cell lymphoma HP:0012190

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
618398 
PMID: 30374066  16/16

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
